README.md

```diff
@@ -58,11 +58,12 @@ For the most up to date information on [UI development browser support](https://
 
 ## Install
 
-<!-- AURO-GENERATED-CONTENT:START (REMOTE:url=https://raw.githubusercontent.com/AlaskaAirlines/WC-Generator/master/componentDocs/partials/usage/componentInstall.md) -->
+<!-- AURO-GENERATED-CONTENT:START (REMOTE:url=https://raw.githubusercontent.com/AlaskaAirlines/WC-Generator/master/componentDocs/partials/usage/componentInstall_esm.md) -->
```

Comment:
I just want to confirm that this component is actually ESM ready since we weren't using that template before.

Reply:
According to this, this component does support ESM:
- https://github.com/orgs/AlaskaAirlines/discussions/502